invokeai/backend/flux/denoise.py

```diff
@@ -40,6 +42,15 @@ def denoise(
         if inpaint_extension is not None:
             img = inpaint_extension.merge_intermediate_latents_with_init_latents(img, t_prev)
 
-        step_callback()
+        step_callback(
+            PipelineIntermediateState(
+                step=step,
+                order=1,
+                total_steps=len(timesteps),
+                timestep=int(t_curr),
+                latents=img,
```

Comment:
Doing something like `latents=old_img - t_curr * pred` would give much better preview images during the early steps. Not sure if this is worth the additional compute/memory overhead though. And maybe users like to see the early noise trajectory - I'm not sure.

Reply:
Updated to this since it made the early preview images look dramatically better and it didn't have a noticeable affect on node execution speed